MICADO/test_micado/test_micado_fits_headers.py

```diff
@@ -0,0 +1,16 @@
+from astropy.io import fits
+import scopesim as sim
+
+
+class TestHeaders:
+    def test_extension_incremental_headers_count_up(self):
+        sim.rc.__config__["!SIM.file.local_packages_path"] = "F:\Work\irdb"
```

Comment:
Fixed paths that refer to your local computer should not be in tests.

Reply:
Done! Thanks for picking that up